src/demo-app/app/shared/media-query-status.ts

```diff
@@ -31,7 +31,9 @@ export class MediaQueryStatus implements OnDestroy {
   private _watcher : Subscription;
   activeMediaQuery : string;
 
-  constructor(media$ : ObservableMedia) { this.watchMediaQueries(media$); }
+  constructor(media$ : ObservableMedia) {
+    this.watchMediaQueries(media$);
```

Comment:
What's the `$` for in the variable name? If it's for private but cannot be marked as such for Angular reasons, we use an underscore to preface the variable